backend/app/tests/test_scraper_system.py

```
@@ -0,0 +1,404 @@
+"""
+Comprehensive test suite for the scraper system.
+Tests scraper functionality, database persistence, and error handling.
+"""
+
+import pytest
+import asyncio
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)